### PR TITLE
Rebalances bluespace beakers

### PR DESCRIPTION
**Pull request by 81Denton**
## About The Pull Request

This PR addresses the two main complaints about bluespace beakers: A) They're cheap to mass print and B) you can easily use them for grenades/elances.

The way I did it is by
A) **Increasing material costs**. 
Diamond and bluespace costs are now 1000u, which is the same diamond cost and twice the bluespace cost of bluespace syringes.
This is before protolathe upgrades and might still be too low, but I'd like to collect some feedback and then later tweak those numbers.

B) **Restricting bluespace beakers to large grenade casings.**
This is effectively a timegate, by locking them behind behind the explosive weaponry techweb node. 
With this change you can no longer use roundstart grenade casings, but first have to research the corresponding techweb node. This will also give large grenade casings a use outside of chemical bomb cores and slime extracts.
Explosive weaponry is a tier 5 node, let me know if you think that it's too obscure and will never be researched.

## Why It's Good For The Game

It's very easy to mass print bluespace beakers and spam elances or hellfoam grenades with them. I think that chemistry is better off if bluespace beakers stay large, but are harder to mass print and can't be used for grenades as easily.

## Changelog
:cl: Denton
balance: Bluespace beakers now only fit in large grenade casings. Large casings have to be researched first and can be printed at the Security and Medical protolathes or techfabs.
balance: Increased the glass (5000), diamond (1000) and bluespace (1000) resource cost for bluespace beakers.
fix: Fixed bluespace beakers only containing glass in their materials list.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/glass.dm [hunk 1]
```diff
@@ -207,7 +207,7 @@
 		and Element Cuban combined with the Compound Pete. Can hold up to \
 		300 units."
 	icon_state = "beakerbluespace"
-	materials = list(MAT_GLASS=3000)
+	materials = list(MAT_GLASS = 5000, MAT_PLASMA = 3000, MAT_DIAMOND = 2000, MAT_BLUESPACE = 1000)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/reagent_containers/glass.dm]: They have more diamond in them than they need to be made

File: code/game/objects/items/grenades/chem_grenade.dm [hunk 2]
```diff
@@ -68,7 +69,10 @@
 			to_chat(user, "<span class='warning'>You need to add an activation mechanism!</span>")
 
 	else if(stage == WIRED && is_type_in_list(I, allowed_containers))
-		. = 1 //no afterattack
+		. = TRUE //no afterattack
+		if(is_type_in_list(I, banned_containers))
+			to_chat(user, "<span class='warning'>[src] is too small to fit [I]!</span>")
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/items/grenades/chem_grenade.dm]:
```suggestion
			to_chat(user, "<span class='warning'>[src] is too small to fit [I]!</span>") // this one hits home huh anon?
```

Comment (DaedalusGame) [reply code/game/objects/items/grenades/chem_grenade.dm]: big use of github beta features

Comment (ExcessiveUseOfCobblestone) [reply code/game/objects/items/grenades/chem_grenade.dm]: This is EXTREMELY IMPORTANT

File: code/modules/reagents/reagent_containers/glass.dm (hunk 1 above)
Comment (81Denton) [reply code/modules/reagents/reagent_containers/glass.dm]: reeee

**Comment by wesoda25** [comment]
Much better than the other PRs

**Comment by optimumtact** [comment]
each of the dplanners is a different person

**Comment by AlexTheSysop42** [comment]
Maybe move explosive weapons behind normal weapons, lock missile launchers behind adv as well if needed

Since both main weapon nodes have 4x cost explosive weaponry is more like ~T13

**Pull request merged**